packages/teuchos/cmake/tpls/FindTPLARPREC.cmake

```diff
@@ -1,58 +1,3 @@
-# @HEADER
```

Comment:
So, are we saying that these tiny `FindTPL<TPLName>.cmake` files don't need copyright and license assertions?  I am okay with that, but I am just wondering what the threshold is for this?

Reply:
So the guidance I have seen indicates that files related to building (e.g., cmake files and build scripts) are not considered source code and therefore do not need them.  Obviously there is some grey region around this.

The other thing to note is that files do not need to have an explicit copyright statement.  The git history implicitly has the needed information, and the default copyright statement and license are in the COPYRIGHT and LICENSE files.  However if someone wants/needs to explicitly add their copyright, they can.  I have taken the path of eliminating as much as possible.  :)

Hope this helps.

Reply:
I guess if the file contains any IP that you care about (i.e. someone took and claimed it as their own you would be upset), you should make sure and assign the copyright and license header.  Otherwise, if it is just a trivial code fragment, who cares.